Pipeline/EMGFilter.m

```diff
@@ -0,0 +1,14 @@
+function [EMGfilt] = EMGFilter(EMGdata,hpvar,lpvar,bworder,samfreq)
```

Comment:
Can you add this feature:
If we put 0 as input for hpvar and lpvar it suppresses the filter.

Reply:
Added the requested feature